Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM node:alpine
+
+ENV NODE_ENV production
```

Comment:
When I ran the `docker-compose build` command the first time, it failed and bailed. Nuxt had an error saying not to use this env variable so I commented this line out with `ENV NODE_ENV production` and it worked.

Any thoughts? I assume it worked for you otherwise you wouldn't have left it in there.